GVFS/GVFS.Common/GVFSPlatform.cs

```diff
@@ -120,13 +120,31 @@ public string GVFSPostIndexChangedHookExecutableName
 
             public string MountExecutableName
             {
-                get { return "GVFS.Mount" + this.ExecutableExtension; }
+                get { return "gvfs.mount" + this.ExecutableExtension; }
             }
 
             public string GVFSUpgraderExecutableName
             {
                 get { return "GVFS.Upgrader" + this.ExecutableExtension;  }
             }
         }
+
+        public class UnderConstructionFlags
+        {
+            public UnderConstructionFlags(
+                bool supportsGVFSService = true,
+                bool supportsGVFSUpgrade = true,
+                bool supportsGVFSConfig = true,
+                bool supportsKernelLogs = true,
+                bool usesGitHooksLoader = false)
+            {
+            }
+
+            public bool SupportsGVFSService { get; }
+            public bool SupportsGVFSUpgrade { get; }
+            public bool SupportsGVFSConfig { get; }
+            public bool SupportsKernelLogs { get; }
+            public bool UsesGitHooksLoader { get; }
```

Comment:
Is `UsesGitHooksLoader` here because it's on the way out in Windows?

Reply:
Yep. See the discussion with Kevin, renaming this to `RequiresDeprecatedGitHooksLoader` to make that more clear.

Reply:
Never mind, I see the other thread on this topic.